pykusto/_src/functions.py

```diff
@@ -1208,7 +1208,7 @@ def dcount(expr: ExpressionType, accuracy: NumberType = None) -> _NumberAggregat
         return _NumberAggregationExpression(KQL(f'dcount({_to_kql(expr)})' if accuracy is None else f'dcount({_to_kql(expr)}, {_to_kql(accuracy)})'))
 
     @staticmethod
-    def dcount_if(expr: ExpressionType, predicate: BooleanType, accuracy: NumberType = 0) -> _NumberAggregationExpression:
+    def dcount_if(expr: ExpressionType, predicate: BooleanType, accuracy: NumberType = 1) -> _NumberAggregationExpression:
```

Comment:
Please see [https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/dcountif-aggfunction](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/dcountif-aggfunction)
The default accuracy is 1, which is more accurate than 0.
I pyKusto should not change the default behavior expected by kusto users working with the web/desktop client